### PR TITLE
Test: Fix Unit tests for BLE MQTT

### PR DESCRIPTION
**Pull request by alfred2g**
Fix Unit tests for BLE MQTT

Description
-----------
Fix Unit tests for BLE MQTT
Checklist:
----------

- [ X ] I have tested my changes. No regression in existing tests.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by ravibhagavandas — APPROVED** [review]

File: tests/unit_test/linux/logging-stack/logging_stack.h [hunk 1]
```diff
@@ -1,182 +0,0 @@
-/*
- * AWS IoT Device SDK for Embedded C V202009.00
```
Comment (aggarw13) [inline tests/unit_test/linux/logging-stack/logging_stack.h:2]: Why has this file been removed? It is purposely added to support logging for unit tests...

**Review by n9wxu — APPROVED** [review]

**Pull request merged**